src/en_us/index.js

```diff
@@ -671,9 +675,13 @@ export const i19triggersMsg = 'List of events that call this procedure'
 export const i19tryAgain = 'Try again'
 export const i19type = 'Type'
 export const i19unable = 'Unable'
+export const i19unableToInstallAppMsg = 'Unable to install app.'
+export const i19unableToUninstallAppMsg = 'Unable to uninstall app.'
```

Comment:
Uai aqui não é um `Msg`, na verdade `unableToInstallApp` é literalmente _Unable to install app_, não?

Reply:
Na verdade o `Msg` é pra quando a tradução é grande demais e você não coloca ela inteira no nome da variável porque não valeria a pena..
Indicando que portanto o nome da variável é uma abreviação, não corresponde exatamente à string retornada.